.gitignore

```
@@ -0,0 +1,4 @@
+env
+.idea
+.env
+
```

Please don't include cache files in the repository.
```suggestion
__pycache__
```